includes/application/forwardApprover.php

```diff
@@ -63,20 +63,38 @@
 	if( isset( $_SESSION['PostName'] ) )
 	{
 		$UID = $_SESSION['PostName'];
+		if ( forwardApprover( $app_id ,$app_type, $UID ) == true )
+		{
+			echo "The application has been successfully FORWARDED to next Post<br>";
+		}
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after closing brace; newline found

<!-- Comment by [SideCI](https://sideci.com) -->